doc/release/1.14.3-notes.rst

```diff
@@ -0,0 +1,43 @@
+==========================
+NumPy 1.14.3 Release Notes
+==========================
+
+This is a bugfix release for a few bugs reported following the 1.14.2 release:
+
+* Printing fixups: stdout of floats in python2, legacy "style" arg for 0d arrays
+* np.lib.recfunctions.fromrecords still accepts a list-of-lists until 1.15
+* PyPy compatibility fix for field titles
+
+The Python versions supported in this release are 2.7 and 3.4 - 3.6. The Python
+3.6 wheels available from PIP are built with Python 3.6.2 and should be
+compatible with all previous versions of Python 3.6. The source releases were
+cythonized with Cython 0.26.1, which is known to **not** support the upcoming
+Python 3.7 release.  People who wish to run Python 3.7 should check out the
+NumPy repo and try building with the, as yet, unreleased master branch of
+Cython.
+
+Contributors
+============
+
+A total of 6 people contributed to this release.  People with a "+" by their
+names contributed a patch for the first time.
+
+* Allan Haldane
+* Charles Harris
+* Jonathan March +
+* Matti Picus
+* Malcolm Smith +
+* Pauli Virtanen
+
+Pull requests merged
+====================
+
+A total of 7 pull requests were merged for this release.
+
+* `#10946 <https://github.com/numpy/numpy/pull/10946>`__: BUG: 'style' arg to array2string broken in legacy mode
+* `#10916 <https://github.com/numpy/numpy/pull/10916>`__: Add missing underscore to prototype in check_embedded_lapack
```

Comment:
I edited the PR title to `BUG: Add missing underscore to prototype in check_embedded_lapack`. I note that you are referencing the originals, I usually reference the backports, which you can get
```
python tools/announce.py $GITHUB v1.14.2..maintenance/1.14.x > doc/changelog/1.14.3-changelog.rst
```
and then add the changelog to git. The file `announce.py` was changed to `changelog.py` in master. You can run it from any branch.

Reply:
Note that it really doesn't matter where the link goes, master or backport, I just find the latter easier to track.